analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/RequestsWithExcessiveLength.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Tests.Diagnostics
+{
+    public class MyController : Controller
+    {
+        [HttpPost]
+        [DisableRequestSizeLimit] // Noncompliant {{Make sure the content length limit is safe here.}}
```

Comment:
All these potentially vulnerable attributes can be used also at the class level. Can we add a test case also for that?